cpp/modules/deck.gl/core/src/lib/view-manager.h

```diff
@@ -37,59 +37,56 @@ class ViewManager {
   ViewManager();
   virtual ~ViewManager();
 
-  // Check if a redraw is needed
+  /// \brief Check if a redraw is needed
   auto getNeedsRedraw(bool clearRedrawFlags = false) -> std::optional<std::string>;
 
-  // Views will be updated deeply (in next animation frame)
+  /// \brief Views will be updated deeply (in next animation frame)
   void setNeedsUpdate(const std::string &reason);
 
-  // Checks each viewport for transition updates
+  /// \brief Checks each viewport for transition updates
   void updateViewStates();
 
-  /** Get a set of viewports for a given width and height
-   * TODO - Intention is for deck.gl to autodeduce width and height and drop
-   * the need for props
-   */
+  /// \brief Get a set of viewports for a given width and height
+  // TODO - Intention is for deck.gl to autodeduce width and height and drop
```

Comment:
In general, format of `TODO` comments should be `// TODO(email): ...`, cpplint should complain if that's not the case. The email can just be the email of the person that wrote the comment, not necessarily that one that should address it.

In this case though, I believe we can get rid of this comment as it refers to `rect` (as noted next to the method signature) parameter which is not part of this implementation

Reply:
Will be removed in next commit